release.json

```diff
@@ -1,6 +1,6 @@
 {
 	"nightly": {
-		"INTEGRATIONS_CORE_VERSION": "master",
+		"INTEGRATIONS_CORE_VERSION": "jaime/pdhfixes",
```

Comment:
Merge this first: https://github.com/DataDog/integrations-core/pull/1183 then switch to `master`